File: Clases 2018/Clases-C#/Simulacro_parcial_2_2018 (creacion de bug e historico en transaccion)/VB_TPS_BugTracker/TPS_BugTracker/BusinessLayer/Services/PrioridadService.vb
Class PrioridadService
    Private oPrioridadDao As PrioridadDao

    Public Sub New()
        oPrioridadDao = New PrioridadDao()
    End Sub

    Public Function consultarPrioridades() As List(Of Prioridad)
        Return oPrioridadDao.getAll()
    End Function
End Class

File: Clases 2018/Clases-C#/ClasePractica11_2018 (update tabla Bugs - asignar bug)/C#_TPS_BugTracker/TPS_BugTracker/BusinessLayer/Services/PrioridadService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TPS_BugTracker.BusinessLayer.Entities;
using TPS_BugTracker.DataLayer.DAOs;

namespace TPS_BugTracker.BusinessLayer.Services
{
    class PrioridadService
    {
        private PrioridadDao oPrioridadDao;
        public PrioridadService()
        {
            oPrioridadDao = new PrioridadDao();
        }

        public List<Prioridad> consultarPrioridades()
        {
            return oPrioridadDao.getAll();
        }
    }
}
